source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,188 @@
+
+SLA stands for Sealed Lead Acid, AGM stands for Absorbent Glass Mat.
+
+The Robot Battery assembly includes a COTS battery, with positive and negative lead cables to an Anderson SB connector.
+The Red/Pink SB50 is STRONGLY recommended.
+
+Always start your match with a fully charged battery - ask for help if it looks like you won't!
+
+Battery Safety & Handling
+-------------------------
+
+Battery assemblies are **delicate**, even though they don't look like it.
+
+**DO NOT CARRY A BATTERY BY THE CABLES**, and always avoid pulling by them.
+Pulling on batteries by the cables may loosen, bend, or break the lug or battery tab -- or even tear the entire tab out of the housing!
+
+.. image:: images/robot-battery/broken-terminal.png
+  :alt: One terminal of an FRC battery fully detached from the battery.
+
+Even if it doesn't get broken, the battery will not be able to provide the same amount of current if the :ref:`connectors are loose <docs/hardware/hardware-basics/preemptive-troubleshooting:Check Battery Connections>`.
+
+//TODO is this a good place to talk about locking down the SB50 so you don't pull on wire, or is that handled on Common troubleshooting?
+
+Dropping the batteries can bend the internal plates, create bulges, or in the worst case - crack the battery case open.
+While we do use AGM [absorbant glass mat] technology for safety, when a cell is punctured it still will leak some liquid.
+This is why FIRST recommends teams have a battery spill kit available. //TODO add link?
+
+Finally, certain older battery chargers can *overcharge* the battery, resulting in boiling off some of the battery acid - another reason for a spill kit.
+
+.. image:: images/robot-battery/boiled.jpg
+  :alt: An FRC battery that was boiled by a high current dumb charger.
+
+Most modern chargers can charge the battery indefinitely without additional harm.
+The picture above is from a high current charger with no maintenance mode that boiled the acid until it produced steam and high pressure that opened the safety vents.
+As such a modern smart charger is highly recommended.
+
+Damaged batteries should be safely disposed of as soon as possible. All retailers that sell large SLA batteries, like car batteries, should be able to dispose of it for you. They may charge a small fee, or provide a small "core charge refund", depending on your state law.
+
+.. warning:: **DO NOT** attempt to "repair" damaged or non-functional batteries.
+
+Battery Construction & Tools
+-----------------------------
+
+Battery Leads
+^^^^^^^^^^^^^^^^^^^
+
+Standard 6AWG, SB50 battery leads normally come in the Kit of Parts, courtesy of AndyMark. Teams are highly encouraged to use these leads.
+
+//TODO cite rules here
+The leads must be copper, minimum size 6AWG and maximum length 12" - there is no maximum size. Teams are recommended to use the standard 6AWG, it is sufficient for almost all robots.
```

Comment:
You say use 6AWG twice despite the 900 article we are thinking about linking to suggests 4AWG wire.  I really think this much specification is overkill for this article.

Reply:
I dialed this back on edit. I want a rookie reading the battery page to know that their 6awg leads are just fine, thank you, and they don't need to go chase 4awg or 2awg performance until they know what they're doing.